x-pack/legacy/plugins/index_lifecycle_management/public/sections/policy_table/components/policy_table/policy_table.js

```diff
@@ -299,6 +299,22 @@ export class PolicyTable extends Component {
     const { name } = policy;
     const cells = Object.entries(COLUMNS).map(([fieldName, { width }]) => {
       const value = policy[fieldName];
+
+      if (fieldName === 'name') {
+        return (
+          <th
+            key={`${fieldName}-${name}`}
+            className="euiTableRowCell"
+            scope="row"
+            data-test-subj={`policyTableCell-${fieldName}`}
+          >
+            <div className={`euiTableCellContent policyTable__content----${fieldName}`}>
```

Comment:
I think `policyTable__content----` contains a typo and should be `policyTable__content--`.

Reply:
good catch, fixed!